companion/src/shared/autowidget.cpp

```
@@ -22,29 +22,106 @@
-AutoWidget::~AutoWidget()
+AutoWidget::~AutoWidget() = default;
+
+void AutoWidget::addBuddyWidget(AutoWidget * wgt)
 {
+  if (wgt) {
+    m_buddyWidgets.append(wgt);
+    // just in case set as we want parent to control
+    clearBuddyBinds(wgt);
+  }
 }
 
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If slots is a macro then please configure it.

(unknownMacro)